examples/hooks-remix/README.md

```diff
@@ -0,0 +1,53 @@
+# Welcome to Remix!
```

Comment:
We should probably replace this with content similar to what we have in other examples READMEs.